src/main/java/school/lemon/changerequest/java/introduction/hw1/Task4.java

```diff
@@ -4,8 +4,15 @@
 public class Task4 {
 
     public static double getSumOfGreatest(double a, double b, double c) {
-        double m1 = Math.max(a,b);
-        double m2 = Math.max(b,c);
-        return m1+m2;
+        if(a > b && b > c)
+            return  a + b;
+        else
+            if(b > a && a > c)
+                return b + a;
+        else
+            if(c > a && a > b)
+            return c + a;
+        else return b + c;
+
```

Comment:
If you use `return` statement in each if clause - there is no need of `else`.
Also it could be simplified, as only 2 if clauses are required in this task.